website/content/docs/commands/auth-methods/update.mdx

```diff
@@ -251,6 +251,21 @@ The following are options are specific to OIDC auth-methods in addition to the c
 
 - `-max-age` `(string: "")` - Indicates the OIDC "max_age" parameter sent to the provider.
 
+- `prompts` `(string: "")` - Indicates whether the OIDC authorization server should display authentication or consent user interface prompts.
+You can optionally configure one or more of the following types of prompts to customize the behavior of the authentication process:
+   - `none` - The authorization server does not display any authentication or consent prompts.
+   - `login` - The authorization server prompts users for reauthentication before allowing them to log in.
+   - `consent` - The authorization server prompts users for consent before returning any information to Boundary.
+   - `select_account` - The authorization server prompts users to select a user account.
+   The `select_account` option can be helpful if your users have multiple sets of login credentials.
```

Comment:
Should we say `multiple accounts` here rather?

Reply:
Yes, that's much cleaner, thanks! I changed it on all 3 pages.